src/main/java/io/confluent/connect/jdbc/source/JdbcSourceTask.java

```diff
@@ -216,6 +235,24 @@ public void start(Map<String, String> properties) {
     running.set(true);
   }
 
+  private List<Map<String, String>> possibleTablePartitions(String table) {
```

Comment:
Please add JavaDoc so that we better explain what this method is doing, especially around what the returned list means/contains.